src/envoy/http/authn/authenticator_base_test.cc

```diff
@@ -20,13 +20,17 @@
 #include "gmock/gmock.h"
 #include "src/envoy/http/authn/test_utils.h"
 #include "test/mocks/network/mocks.h"
+//#include "test/mocks/request_info/mocks.h"
```

Comment:
if not needed, please remove it, don't just comment it out

Reply:
Done